store.go

```diff
@@ -2479,13 +2479,29 @@ func (s *store) Mount(id, mountLabel string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+
+	s.graphLock.Lock()
```

Comment:
Can we move this down to line 2493 and immediately unlock as soon as we don't need? Deferring is comfortable but we could try to keep the critical sections as short as possible - if possible :)

Reply:
that could cause a deadlock.  The locks taken on the stores are always taken after the graphLock